eox-core-v-1-0/prose/edit/src/design-considerations-01-construction-principles.md

```diff
@@ -1,10 +1,32 @@
 ## Construction Principles
 
-TODO:
+An End-of-X (EOX) lifecycle statement represented by an OpenEoX Core document is created through the application of
+well-defined schemas during a process of collaboration between multiple stakeholders including vendors,
+Product Security Incident Response Teams (PSIRTs), and lifecycle management authorities.
```

Comment:
Well: usually, the product manager/ vendor decides that - there is usually not a collaboration between multiple stakeholders, neither do we have multiple schemas in the core section.